sql-ballerina/tests/local-transaction-test.bal

```diff
@@ -62,9 +62,9 @@ function testLocalTransaction() {
     boolean committedBlockExecuted = false;
     transactions:Info transInfo;
     retry<SQLDefaultRetryManager>(1) transaction {
-        var res = dbClient->execute("Insert into Customers (firstName,lastName,registrationID,creditLimit,country) " +
+        var res = checkpanic dbClient->execute("Insert into Customers (firstName,lastName,registrationID,creditLimit,country) " +
```

Comment:
Can we use `check` instead of `checkpanic`?
```suggestion
        var res = check dbClient->execute("Insert into Customers (firstName,lastName,registrationID,creditLimit,country) " +
```
Please check all the places.